SettingsToJson.py

```diff
@@ -117,7 +117,7 @@ def GetSettingJson(setting, web_version, as_array=False):
 
         tags_list = []
 
-        for option_name in setting_info.choice_list:
+        for option_name in [o for o in setting_info.choice_list if o not in settingJson.get('option_remove', [])]:
```

Comment:
Would be cleaner to just `if o in settingJson.get('option_remove', []): continue` in the loop imo.

Reply:
Yeah, I suppose so. Done.